modules/core/src/main/java/org/apache/ignite/mxbean/IgniteMXBean.java

```diff
@@ -27,36 +27,152 @@
+
+    /** */
+    public String ATTR_DESC_EXECUTOR_SRVC_FORMATTED = "Formatted instance of fully configured thread pool that is used in grid.";
+
+    /** */
+    public String ATTR_DESC_IGNITE_HOME = "Ignite installation home folder.";
+
+    /** */
+    public String ATTR_DESC_MBEAN_SERVER_FORMATTED = "Formatted instance of MBean server instance.";
+
+    /** */
+    public String ATTR_DESC_LOC_NODE_ID = "Unique identifier for this node within grid.";
+
+    /** */
+    public String ATTR_DESC_IS_PEER_CLS_LOADING_ENABLED = "Whether or not peer class loading (a.k.a. P2P class loading) is enabled.";
+
+    /** */
+    public String ATTR_DESC_LIFECYCLE_BEANS_FORMATTED = "String representation of lifecycle beans.";
+
+    /** */
+    public String ATTR_DESC_ACTIVE = "Checks Ignite grid is active or is not active.";
+
+    /** */
+    public String ATTR_DESC_DISCOVERY_SPI_FORMATTED = "Formatted instance of configured discovery SPI implementation.";
+
+    /** */
+    public String ATTR_DESC_COMMUNICATION_SPI_FORMATTED = "Formatted instance of fully configured SPI communication implementation.";
+
+    /** */
+    public String ATTR_DESC_DEPLOYMENT_SPI_FORMATTED = "Formatted instance of fully configured deployment SPI implementation.";
+
+    /** */
+    public String ATTR_DESC_CHECKPOINT_SPI_FORMATTED = "Formatted instance of configured checkpoint SPI implementation.";
+
+    /** */
+    public String ATTR_DESC_COLLISION_SPI_FORMATTED = "Formatted instance of configured collision SPI implementations.";
+
+    /** */
+    public String ATTR_DESC_EVT_STORAGE_SPI_FORMATTED = "Formatted instance of fully configured event SPI implementation.";
+
+    /** */
+    public String ATTR_DESC_FAILOVER_SPI_FORMATTED = "Formatted instance of fully configured failover SPI implementations.";
+
+    /** */
+    public String ATTR_DESC_LOAD_BALANCING_SPI_FORMATTED = "Formatted instance of fully configured load balancing SPI implementations.";
+
+    /** */
+    public String ATTR_DESC_OS_INFO = "OS information.";
+
+    /** */
+    public String ATTR_DESC_JDK_INFO = "JDK information.";
+
+    /** */
+    public String ATTR_DESC_OS_USER = "OS user name.";
+
+    /** */
+    public String ATTR_DESC_VM_NAME = "VM name.";
+
+    /** */
+    public String ATTR_DESC_INSTANCE_NAME = "Optional kernal instance name.";
+
+    /** */
+    public String ATTR_DESC_CUR_COORDINATOR_FORMATTED = "Formatted properties of current coordinator.";
+
+    /** */
+    public String ATTR_DESC_IS_NODE_BASELINE = "Baseline node flag.";
+
+    /** */
+    public String ATTR_DESC_READ_ONLY_MODE = "Cluster read-only mode status.";
+
+    /** */
+    public String ATTR_DESC_READ_ONLY_MODE_DURATION = "Duration of read-only mode enabled on cluster.";
+
     /**
      * Gets string presentation of the version.
      *
      * @return String presentation of the version.
+     * @deprecated Use metric API.
```

Comment:
Let's replace this and all others deprecation comments with the comment that equal all over the code:\

` * @deprecated Use {@link GridMetricManager} instead.`

Reply:
done